srcs/juloo.keyboard2/SettingsActivity.java

```diff
@@ -30,12 +30,27 @@ public void onCreate(Bundle savedInstanceState)
       system is using light theme. */
   void detectSystemTheme()
   {
-    if (Build.VERSION.SDK_INT >= 14)
+    if (Build.VERSION.SDK_INT >= 14 && Build.VERSION.SDK_INT <= 20)
     {
       int ui_mode = getResources().getConfiguration().uiMode;
-      if ((ui_mode & Configuration.UI_MODE_NIGHT_NO) != 0)
-        setTheme(android.R.style.Theme_DeviceDefault_Light);
-    }
+   
+   if ((ui_mode & Configuration.UI_MODE_NIGHT_NO) != 0) {
+setTheme(android.R.style.Theme_Holo_Light);
+}else{
+setTheme(android.R.style.Theme_Holo
+}
+
+
+    }else if (Build.VERSION.SDK_INT >= 21)
+    {
+
+int ui_mode = getResources().getConfiguration().uiMode;
+   
+   if ((ui_mode & Configuration.UI_MODE_NIGHT_NO) != 0) {
+setTheme(R.style.AppTheme);
+}else{
+setTheme(android.R.style.Theme_Material
+}}
```

Comment:
This contains some code duplication and syntax errors.

Reply:
:v